metashare/api/models.py

```diff
@@ -138,7 +138,7 @@ class ProjectSlug(AbstractSlug):
 class Project(mixins.HashIdMixin, mixins.TimestampsMixin, SlugMixin, models.Model):
     name = models.CharField(max_length=50, unique=True)
     description = MarkdownField(blank=True, property_suffix="_markdown")
-    pr_url = models.URLField(unique=True)
+    pr_url = models.URLField(unique=True, null=True)
     commit_message = MarkdownField(blank=True, property_suffix="_markdown")
     release_notes = MarkdownField(blank=True, property_suffix="_markdown")
 
```

Comment:
@wlonk We are asking for just the `name` and `description` field for initial project creation, so I think the other fields can also be null.

Reply:
@duggiemitchell `blank=True` should account for that; I'd rather that they be empty strings than null, unless null means something distinct from an empty string.

Reply:
@wlonk I don't think the front end should need to know to add in keys with empty values to the `POST` data. We're using a form, so IMO the back end should be able to handle a `POST` that only includes the data that the user or front end needs to provide, e.g.:

```
{
  'name': 'My Project',
  'description': '',
}
```

Reply:
Right; so missing fields + defaults should handle that.

Reply:
(That is, defaults on the backend.)